extensions/amp-date-picker/validator-amp-date-picker.protoascii

```diff
@@ -167,13 +167,15 @@ attr_lists: {
   attrs: {
     name: "src"
     value_url: {
+      protocol: "http"
       protocol: "https"
       allow_relative: false
```

Comment:
If allowing `http` why keep `allow_relative` as `false`?

Reply:
I was basing this off of some of the social components, but I notice that amp-story does what you're suggesting for a more similar use case. It makes sense now that relative is bad for a social extension because the content is always on a different domain.

Reply:
After Ali's comment below, I'll remove the `http` protocol